main/templates/tutorial/run.md

```diff
@@ -1,11 +1,13 @@
 We assume that you already know how to run the Google App Engine
 applications from a command line, but if not you should refer to the
 [documentation](https://developers.google.com/appengine/docs/python/gettingstartedpython27/helloworld).
-Execute in your terminal the `run.py` from the root directory
+
+Since gae-init uses Gulp to build assets and bower to manage JavaScript libraries, you'll need to run
+bower to get started:
```

Comment:
Since you should be already aware of what is going on with gae-init.. I would change this to simply:

> Execute in your terminal the following commands from the root directory: